ProfessorSpace/src/model/Question.java

```diff
@@ -0,0 +1,69 @@
+package model;
+
+public class Question {
+	
+	private String description;
+	private String option1;
```

Comment:
It is better to replace with an array operation other than define option1 to option4. It will be easy to mofiy

Reply:
This model will not be handled by me after I changed the directory to under the project. I will remove this file. Thanks.